include/lattices/priority_lattice.hpp

```diff
@@ -22,7 +22,7 @@ struct PriorityValuePair {
   P priority;
   V value;
 
-  PriorityValuePair(P p = 0, V v = {}) : priority(p), value(v) {}
+  PriorityValuePair(P p = 1000000, V v = {}) : priority(p), value(v) {}
```

Comment:
Shouldn't this be ÌNTMAX` or the equivalent in C++?

Reply:
In principle it should be P_MAX but obviously we don't have that...The robot code is actually using "double" for example. But for now let's go with INT_MAX?